tools/rapids-package-name

```diff
@@ -48,21 +55,23 @@ if (( append_wheelname )) && [[ -v RAPIDS_PY_WHEEL_NAME ]] && [[ "${RAPIDS_PY_WH
   pkg_name+="_${RAPIDS_PY_WHEEL_NAME}"
 fi
 
-# for python package types, add pyver
+# for python package types (except pure wheels), add pyver
 if (( append_pyver == 1 )); then
   pkg_name+="_${RAPIDS_PY_VERSION//./}"
```

Comment:
Could we also modify this to include "py"? This was requested recently by @jakirkham and seems like a reasonable change to me.
```suggestion
  pkg_name+="_py${RAPIDS_PY_VERSION//./}"
```

Reply:
Yeah that is fine with me.